### DISCUSSION TITLE
Do all fleet adapters share the same graph?

### DISCUSSION THREAD

**Discussion by Achllle** [osrf/rmf_core#313] [Q&A]
In the situation where there are several fleets **partially** sharing the same space and fleets are restricted to certain areas (common in industrial manufacturing), do each of these fleets need to share the same navigation graph? If they do, is the bidding/negotation system responsible for ensuring that these robots don't leave their working areas, even when a traffic conflict arises?

If they don't share the same graph, how is that overlap coordinated? I'm not clear on this from looking through the code and documentation.

**Comment by gbiggs** [comment]
Thanks for your question. However we ask that you please ask questions on https://github.com/open-rmf/rmf/discussions following our new support guidelines: https://github.com/open-rmf/rmf/wiki/Support-guidelines

**Reply by Achllle** [reply]
Sure, if I can just point out that this transition isn't indicated anywhere on this repository. Probably a good idea to indicate here on discussions or on the main README or even archive this repo.

**Comment by gbiggs** [comment]
Yes, it's still in progress.
